CoinDock-api/database/migrations/2022_05_17_161749_create_coins_table.php

```diff
@@ -15,7 +15,11 @@ public function up()
     {
         Schema::create('coins', function (Blueprint $table) {
             $table->id();
+            $table->string('coin_id');
```

Comment:
Don't update existing migrations. If any new changes were required then introduce a separate migration for the  changes.